### PR TITLE
Ajout mentions légales

### LINKED ISSUE #505: Mentions légales

### PR DESCRIPTION
**Pull request by tmerlier**
Fix #505

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     },
 
     "Footer": {
-      "terms": "Conditions générales d’utilisation"
+      "terms": "Mentions légales"
```
Comment (tusbar) [inline src/locales/fr.json]: `"legal": "Mentions légales"`

File: src/locales/en.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     },
 
     "Footer": {
-      "terms": "Terms of Service"
+      "terms": "Legal"
```
Comment (tusbar) [inline src/locales/en.json]: `"legal": "Legal"`

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -19,7 +20,9 @@ const Footer = ({ t }) => (
       <div className={styles.footerLogo}>
         <ul className={styles.footerLinks}>
           <li><h2>geo.data.gouv.fr</h2></li>
-          <li>{t('components.Footer.terms')}</li>
+          <li>
+            <Link to='/legal'>{t('components.Footer.terms')}</Link>
```
Comment (tusbar) [inline src/components/Footer/Footer.js]: `<Link to='/legal'>{t('components.Footer.legal')}</Link>`

File: src/routes/Legal/components/LegalPage/legalPage.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.container {
```
Comment (tusbar) [inline src/routes/Legal/components/LegalPage/legalPage.scss]: this file should be called `LegalPage.scss`

File: src/routes/Legal/containers/LegalContainer.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+import LegalPage from '../components/LegalPage'
+
+export default () => (
+  <LegalPage />
+)
```
Comment (tusbar) [inline src/routes/Legal/containers/LegalContainer.js]: This is not a container, and it is not needed.

**Review by tusbar — CHANGES_REQUESTED** [review]

**Comment by tusbar** [comment]
@tmerlier tu peux rebase ?

**Review by jdesboeufs — APPROVED** [review]

**Pull request merged**